src/datatypes/uncertainty.coffee

```diff
@@ -11,7 +11,11 @@ module.exports.Uncertainty = class Uncertainty
         # Uncertainties with different types probably shouldn't be supported.
         return false
       if typeof a.after is 'function' then a.after b else a > b
+    isNonEnumerable = (val) ->
+      val? and (val.isCode or val.isConcept or val.isValueSet)
     if typeof @high is 'undefined' then @high = @low
+    if isNonEnumerable(@low) then @low = null
+    if isNonEnumerable(@high) then @high = null 
```

Comment:
I hadn't thought of this before, but your unit tests made me think of it.  This could end up generating something like: `Uncertainty(1, null)`.  This concerns me because in the `lessThan` implementation (which everything else drives off of), `null` appears to have special meaning (it seems to be neg/pos infinity):
https://github.com/cqframework/cql-execution/blob/19775fd74c97c5d054e69f5aa6a050d9bb4fe450/src/datatypes/uncertainty.coffee#L48-L55

We don't want a code to be misinterpreted as an infinity.  I think it might be better that if either `@low` or `@high` are not enumerable, then we set _both_ to `null`.  In that case, I think `lessThan` will always return null.

What do you think of that?